website/server/controllers/top-level/dataexport.js

```diff
@@ -97,7 +112,26 @@ async function _getUserDataForExport (user) {
  * @apiGroup DataExport
  * @apiDescription NOTE: Part of the private API that may change at any time.
  *
- * @apiSuccess {String} A json file
```

Comment:
That's true. For this route (and the XML one) I'm comfortable with just describing it and not including extensive information about it.